### PR TITLE
"xml.format.enforceQuoteStyle"="ignore" is default

### LINKED ISSUE #263: Set "xml.format.enforceQuoteStyle"="ignore" by default
## Input (valid XML)
```xml
<foo bar='Hello "world"!' />
```

## Formatted output (with `doubleQuotes` preference -- produces invalid XML)
```xml
<foo bar="Hello "world"!" />
```

## Expected output (valid XML -- quotes preference should be ignored in favor of keeping the XML valid)
```xml
<foo bar='Hello "world"!' />
```

### PR DESCRIPTION
**Pull request by datho7561**
Modified README to reflect this as well. Partially closes #263

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

**Comment by datho7561** [comment]
Peek 2020-05-25 11-51

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The following settings are supported:
 * `xml.symbols.excluded`: Disable document symbols (Outline) for the given file name patterns. Updating file name patterns does not automatically reload the Outline view for the relevant file(s). Each file must either be reopened or changed, in order to trigger an Outline view reload.
 
 Since 0.12.0:
-* `xml.format.enforceQuoteStyle`: preferred|ignore : Enforce preferred quote style / ignore quote formatting.
+* `xml.format.enforceQuoteStyle`: preferred|ignore : Enforce preferred quote style / ignore quote formatting. Default is `ignore`.
```
Comment (xorye) [inline README.md:80]: I know you didn't do this (in fact, I did) but I think we can replace this line with 
* `xml.format.enforceQuoteStyle`: Enforce `preferred` quote style (set by `xml.preferences.quoteStyle`) or `ignore` quote style when formatting. Default is `ignore`.

@fbricon @angelozerr what do you think?

File: package.json [hunk 2]
```diff
@@ -181,7 +181,7 @@
             "Use preferred quote style set by `xml.preferences.quoteStyle` when formatting.",
             "Ignore quote style formatting."
           ],
-          "default": "preferred",
+          "default": "ignore",
           "description": "Set quote style rule when formatting.",
```
Comment (xorye) [inline package.json:185]: If the new description from the README is okay, we could probably use the same description here too

File: README.md (hunk 1 above)
Comment (fbricon) [reply README.md:80]: +1

File: package.json (hunk 2 above)
Comment (fbricon) [reply package.json:185]: +1

**Comment by angelozerr** [comment]
Can we merge this PR?

**Comment by datho7561** [comment]
I implemented the changes that David suggested, and I think it's ready to merge. On May 27, 2020 7:27 AM, Angelo <notifications@github.com> wrote:
Can we merge this PR?

**Pull request merged**